src/ext/repository.rs

```diff
@@ -330,4 +338,12 @@ impl RepositoryExtended for gix::Repository {
                 }
             })
     }
+
+    fn git_data_file(&self, path: &str) -> PathBuf {
+        // If STG_EDIT_IN_CWD is set, return path as is.
+        match std::env::var("STG_EDIT_IN_CWD") {
```

Comment:
Is there known cases where someone might need to fallback to the old behavior?

If not, I'd prefer we not have this environment variable at all; keep it simple.

Another issue with this env var is that we're only checking its existence, not its value. So if someone set `STG_EDIT_IN_CWD=0`, for example, it would fallback to the old behavior the same as if they'd set `STG_EDIT_IN_CWD=1`.